import java.util.InputMismatchException;
import java.util.Scanner;

public class MultipleException {
    public static void main(String[] args){
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter the input: ");
        try {
            int userInput = scanner.nextInt();
            int result = 10/userInput;
            System.out.println("The result is: " + result);
        }catch (ArithmeticException e){
            System.out.println("Exception: " + e.fillInStackTrace());
        }catch (InputMismatchException e){
            System.out.println("Exception: " + e.fillInStackTrace());
        }finally {
            scanner.close();
        }
        System.out.println("code after the try catch block");
    }
}
